src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XsltArgumentList.cs

```diff
@@ -4705,15 +4717,30 @@ public void NodeIter2()
                 xslArg.AddParam("sourceUri", String.Empty, uriSource.ToString());
 
                 xslt.Load(FullFilePath("xsd2cs2.xsl"), new XsltSettings(true, true), new XmlUrlResolver());
-
-                XPathDocument doc = new XPathDocument(FullFilePath("sample.xsd"));
-                StringWriter sw = new StringWriter();
-                xslt.Transform(doc, xslArg, sw);
-                sw.Dispose();
-                return;
             });
 
             Assert.Equal("Compiling JScript/CSharp scripts is not supported", e.InnerException.Message);
```

Comment:
We should not verify the messages in exceptions because this will break if we localize. Unless someone does the work to allow tests to pull resources from arbitrary places.
I know there's ~100 cases where we already do and you didn't add this but just FYI in future.